docs/testing-guidelines/CodeCoverageAnalysis.md

```diff
@@ -0,0 +1,136 @@
+# Code coverage analysis for commit [2ae5d07](https://codecov.io/gh/PowerShell/PowerShell/tree/c7b959bd6e5356fbbd395f22ba0c6cba49f354f6/src)
+
+Code coverage runs are enabled on daily Windows builds for PowerShell 6.0. The results of the latest build are available at: [CodeCov.io](https://codecov.io/gh/PowerShell/PowerShell)
```

Comment:
Also use semantic line breaks (each sentence on own line, makes it easier to see diffs)